mobile/app/src/main/res/values/arrays.xml

```diff
@@ -63,4 +74,26 @@
         <item>7</item>
         <item>14</item>
     </string-array>
+
+    <string-array name="pref_languages_titles">
+        <item>Czech</item>
+        <item>English</item>
+        <item>French</item>
+        <item>German</item>
+        <item>Japanese</item>
+        <item>Portuguese (Brazil)</item>
+        <item>Russian</item>
+        <item>Spanish (Latin America)</item>
+    </string-array>
+
+    <string-array name="pref_languages_values">
+        <item>cs</item>
+        <item>en</item>
+        <item>fr</item>
+        <item>de</item>
+        <item>ja</item>
+        <item>pt_BR</item>
+        <item>ru</item>
+        <item>es_419</item>
+    </string-array>
```

Comment:
Seems that you are missing Spanish (es) :wink:

Reply:
Unfortunately, translation into Spanish is now only 25% complete.
This list will be updated as translations are ready

Reply:
Oops my fault, I was looking at the translations for the "Button" :see_no_evil: